storage-proofs/src/layered_drgporep.rs

```diff
@@ -428,6 +445,54 @@ pub trait Layers {
                             // If we panic anywhere in this closure, thread.join() below will receive an error —
                             // so it is safe to unwrap.
                             let graph = transfer_rx.recv().unwrap();
+
```

Comment:
it would be nicer to just call a function here e.g `let tree_d = graph.merkle_tree()` and have that be different depending on the compile flag

Reply:
Yes, definitely much nicer.